_includes/header.html

```diff
@@ -6,7 +6,7 @@
           This site is in alpha and a <a href="https://playbook.cio.gov" target="_blank">work in progress</a>
         </div>
         <img class="eop-seal" alt="United States flag icon" src="{{ site.github.url }}/img/layout/usa-flag.png" />
-        <span class="hidden-xs">An official website of the </span>United States Government
+        <span class="hidden-xs">An official website of the </span>Department of Homeland Security
```

Comment:
I know DHS.gov has this on their website, but honestly the convention is to use the United States Government version. That's what I'd stick with.